### PR TITLE
fix(lint): Exclude event triggered subprocesses from no-disconnected rule

### LINKED ISSUE #31: Exclude Event Sub-Process From No-Disconnected Rule
We have a [`no-disconnected` rule](https://github.com/bpmn-io/bpmnlint/blob/master/rules/no-disconnected.js) that checks for flow elements that are not connected to other flow elements (usually an error).

The rule seems to apply to [event sub-processes](https://docs.camunda.org/manual/7.5/reference/bpmn20/subprocesses/event-subprocess/), too:

foo

Those, by definition are never connected to other flow elements and should ignored in the check.

__Describe the solution you'd like__

* [ ] The linter excludes event sub-processes when checking for the `no-disconnected` rule

### PR DESCRIPTION
**Pull request by oguzeroglu**
Fixes #31 

By definition, it does not make sense for event triggered sub events to be check with no-disconnected rule:

Screenshot 2019-09-02 at 15 12 04

### CODE REVIEW

File: rules/no-disconnected.js [hunk 1]
```diff
@@ -17,7 +18,7 @@ module.exports = function() {
       'bpmn:Gateway',
       'bpmn:SubProcess',
       'bpmn:Event'
-    ])) {
+    ]) || (is(node, 'bpmn:SubProcess') && node.triggeredByEvent)) {
```
Comment (nikku) [inline rules/no-disconnected.js]: Can't we simplify this? 

```javascript
if (!isAny(node, [
      'bpmn:Task',
      'bpmn:Gateway',
      'bpmn:SubProcess',
      'bpmn:Event'
    ]) || node.triggeredByEvent) {
```

Comment (nikku) [reply rules/no-disconnected.js]: Could you add test coverage for this case, too?

**Comment by oguzeroglu** [comment]
@nikku Done!

**Comment by philippfromme** [comment]
Make sure to clean up the commit message when squashing. 👍 

image

**Comment by oguzeroglu** [comment]
@philippfromme Thanks! Done 👍

**Comment by philippfromme** [comment]
You can re-request @nikku's review to let him know it's ready to be reviewed again. 💪

**Comment by pinussilvestrus** [comment]
Congratulations on your first solved issue @oguzeroglu 👍

**Comment by oguzeroglu** [comment]
@pinussilvestrus Thanks!

**Pull request merged**